src/config.py

```diff
@@ -157,6 +157,11 @@ def __init__(self, print_warnings: bool = False):
         if "barbarian" in self._custom:
             self.barbarian.update(self._custom["barbarian"])
 
+        # Necro config
+        self.necro = self._config["necro"]
+        if "necro" in self._custom:
+            self.necro.update(self._custom["necro"])
```

Comment:
With the clear_pindle_pack being a bool and not a string you have to type cast it here:
`self.necro["clear_pindle_pack"] = bool(int(self.necro["clear_pindle_pack"]))`

Reply:
Added a cast in the necro class, since I'm using the hotkey config (which are strings) it didnt make sense to deal with casting all of them.
The global configs all get casted, but the fX keys just stay strings it looks like.
I could move this to the global config but since its only used by the necro that didnt make sense either.

Reply:
you could leave it as is. So everything is strings, and then only add the line I posted, which will only cast the one field. But fine like this for now.